packages/runtime/src/account.ts

```diff
@@ -227,6 +235,22 @@ export class StoreAccount implements StoreAccountI {
     }
   }
 
+  // opt-in to asset
+  optInToASA (assetIndex: number, assetHolding: AssetHolding): void {
+    const localAsset = this.assets.get(assetIndex); // fetch asset holding of account
```

Comment:
how about renaming to `localAsset` to `assetHolding` ?

Reply:
renamed